### ISSUE TITLE
OutlookMessage.getClientSubmitTime produces NullPointerException

### ISSUE THREAD

**Issue by Faelean** [bbottema/simple-java-mail#243]
While investigating https://github.com/bbottema/outlook-message-parser/issues/23 we found an issue with the OutlookMessage.getClientSubmitTime() method.

```
java.lang.NullPointerException
	at org.simplejavamail.outlookmessageparser.model.OutlookMessage.getClientSubmitTime(OutlookMessage.java:849)
	at org.simplejavamail.internal.outlooksupport.converter.OutlookEmailConverter.buildEmailFromOutlookMessage(OutlookEmailConverter.java:73)
	at org.simplejavamail.internal.outlooksupport.converter.OutlookEmailConverter.outlookMsgToEmailBuilder(OutlookEmailConverter.java:60)
	at org.simplejavamail.converter.EmailConverter.outlookMsgToEmailBuilder(EmailConverter.java:226)
	at org.simplejavamail.converter.EmailConverter.outlookMsgToEmail(EmailConverter.java:209)
	at org.simplejavamail.converter.EmailConverter.outlookMsgToEmail(EmailConverter.java:201)
```

The issue is present with the following test emails from the outlook-message-parser.

- tst_unicode.msg
- unsent draft.msg
- chinese message.msg
- forward with attachments and embedded images.msg
- the nested mail from 'nested simple mail.msg'

**Comment by bbottema** [comment]
Resolved in https://github.com/bbottema/outlook-message-parser/issues/25

**Comment by bbottema** [comment]
Released in 6.0.0
